src/eko/matching_conditions/__init__.py

```diff
@@ -97,11 +97,11 @@ def split_ad_to_evol_map(
                         {
                             f"{hq}+.S": ome_members["S_Hq"],
                             f"{hq}+.g": ome_members["S_Hg"],
```

Comment:
Intrinsic is when the input is intrinsic, so the second one is an H. If the first one is an H it's fine, and it should be present even if there was no intrinsic in the beginning, because it is some light flavor / the gluon that generates the (was-)heavy flavor.

Reply:
Furthermore `f"T{n}.S"` has to be removed, as we agreed to match in the (g, S, V, ..., c+, c-) -> (g, S, V, ..., c+, c-) basis. Then there will be a `f"{hq}+.S" = ome_members["S_hq"]` there, for example.

Reply:
sure this is fixed.